.github/workflows/e2e_user_features.yml

```diff
@@ -0,0 +1,53 @@
+name: End-to-End tests
```

Comment:
Should this be User features e2e tests

Reply:
No, all the e2e workflow files have this name. It makes filtering easier